worlds/ahit/docs/en_A Hat in Time.md

```diff
@@ -0,0 +1,51 @@
+# A Hat in Time
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Items which the player would normally acquire throughout the game have been moved around. 
```

Comment:
per the [style guide](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md#markdown) markdown should be 120 chars per line, and this includes game docs. it's fine to follow your own style, but this is inconsistent.

Reply:
Done